### ISSUE TITLE
WCAG and HTML validation

### ISSUE THREAD

**Issue by SidraShahid0510** [SidraShahid0510/auction-house-semester-project#9]
- HTML validation of all following pages through https://validator.w3.org/ and cross testing of website in different browsers.
- The site should be WCAG compliant.
- Use the WAVE Web Accessibility Evaluation Tools to test that site matches best practices for accessibility.
